util/tune/current_settings.rb

```diff
@@ -216,15 +220,12 @@ def puppetdb_java_args
 # TODO: update to eliminate the use of exclusions
 #   See https://tickets.puppetlabs.com/browse/SLV-531
 #
-def get_conf_parameter(file, parameter, exclusions = DEFAULT_EXCLUSION)
+def get_conf_parameter(file, parameter)
```

Comment:
Remove `exclusions` from the YARD doc for this method.

Reply:
done